FWCore/Framework/src/Worker.h

```diff
@@ -645,13 +618,19 @@ namespace edm {
       typedef OccurrenceTraits<EventPrincipal, BranchActionStreamBegin> Arg;
       static bool call(Worker* iWorker,
                        StreamID,
-                       EventPrincipal const& ep,
-                       EventSetupImpl const& es,
+                       EventTransitionInfo const& info,
                        ActivityRegistry* /* actReg */,
                        ModuleCallingContext const* mcc,
                        Arg::Context const* /* context*/) {
         //Signal sentry is handled by the module
-        return iWorker->implDo(ep, es, mcc);
+        return iWorker->implDo(info.principal(), info.eventSetupImpl(), mcc);
+      }
+      static void esPrefetch(Worker* worker,
```

Comment:
Please add `Async` to the name to make it clear this is not a synchronous call.

Reply:
Thanks. That is better. I am changing this now and also all the similar places referenced by the the other comments. I will push the commit soon.